app/controllers/petitions_controller.rb

```diff
@@ -30,6 +30,8 @@ def show
   def create
     params[:petition][:creator_signature_attributes][:humanity] = Captcha.verify(params[:captcha_response_field], params[:captcha_string])
 
+    set_duration_to_fixed_date_flag!
+
```

Comment:
We don't want this in. This was the automatic pre-purdah end a petition on a set date 30/03/2015